packaging/dependencies.nix

```diff
@@ -63,6 +63,7 @@ scope: {
         "--with-coroutine"
         "--with-iostreams"
       ];
+      enableIcu = false;
```

Comment:
We could in a different PR probably use disallowedReferences to make sure we never reference to icu in nix.